package.json

```diff
@@ -533,7 +533,7 @@
 				},
 				{
 					"command": "forge.translateObject",
-					"when": "view == forgeDataManagementView && viewItem == object",
+					"when": "viewItem == object || viewItem == version",
```

Comment:
Could we keep the filtering by view? I understand that the command may now appear in two different views but I wouldn't solve that by simply removing the `view === ...` part. There could be other views with nodes of type `version` (for example, in the Design Automation tree), and we don't want the command to show up for those.

Reply:
I tried originally but I didn't succeed in having multiple and/or conditions. I can spend more time trying to find the way to express more complex when conditions.

Reply:
Ok found a way by having 2 command definitions with different when clauses.